src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  max-width: 1024px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 11th item. Don't hardcode max-width, you can calculate it tho if you would have variables for card-size, gap between them and cards in a row count